timesketch/lib/analyzers/base_sessionizer_test.py

```diff
@@ -208,3 +207,33 @@ def _create_eventObj(datastore, event_id, ts, source_attrs=None):
 
     datastore.import_event(event['_index'], event['_type'], event['_source'],
                            str(event_id))
+
+
+def check_surrounding_events(TestClass, datastore, threshold_ids,
```

Comment:
```suggestion
def check_surrounding_events(test_instance, datastore, threshold_ids,
```

And rename the variable name in the function below.